assets/dev/scss/common/common.scss

```diff
@@ -2,12 +2,15 @@
 @import "../../../../assets/dev/scss/helpers/variables";
 @import "../../../../assets/dev/scss/editor/breakpoints";
 @import "../global/global";
+@import "../global/dialog-base";
```

Comment:
I wasn't aware to the difference between `elementorFrontend` and `elementorCommon` localization object.
I have removed the conditional loading from the situation in which the user has the WordPress admin bar displayed in the frontend.